agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/ConfigurationBuilder.java

```diff
@@ -478,7 +479,9 @@ private static Configuration loadConfigurationFile(Path agentJarPath) {
       return getConfiguration(runtimeAttachedConfigurationContent, JsonOrigin.RUNTIME_ATTACHED);
     }
 
-    if (SdkVersionPrefixHolder.isRpIntegration()) {
+    // any manual RP integration will support loading user's applcationinsights.json
+    if (SdkVersionPrefixHolder.isRpIntegration()
+        && RpAttachType.getRpAttachType() != RpAttachType.MANUAL) {
```

Comment:
would this work?
```suggestion
    if (RpAttachType.getRpAttachType() == RpAttachType.AUTO) {
```

Reply:
latest exporter has 3 types 

MANUAL("Manual"), // Manually writing code to start collecting telemetry, e.g. via azure-monitor-opentelemetry-exporter
  STANDALONE_AUTO("StandaloneAuto"), // RP attach is enabled via a custom JAVA_OPTS
  INTEGRATED_AUTO("IntegratedAuto"); // RP attach is on by default

Reply:
beta.12 will have this..  no config is allowed for non MANUAL

Reply:
how about then?
```suggestion
    if (RpAttachType.getRpAttachType() != RpAttachType.MANUAL) {
```

Reply:
getRpAttachType will run into assertError in the non-attach scenario setRpAttachType is never called

Reply:
getRpAttachType will return null in that case if we don't throw assertError

Reply:
we need to check both

Reply:
> getRpAttachType will run into assertError in the non-attach scenario setRpAttachType is never called

can we set it in all cases?